CHANGELOG.md

```diff
@@ -1,8 +1,57 @@
 ## dbt 0.21.0 (Release TBD)
 
+## dbt 0.21.RC1 (September 17, 2021)
```

Comment:
The release process auto-updates this so you don't have to do it here. If we just remove this line then the GitHub Action will do it for us